docs/source/glossary/glossary.md

```diff
@@ -41,17 +41,23 @@ The data manifest is a file that includes the locations of all fields of view fo
 A collection of Image Tiles corresponding to a specific volume or plane of the sample, under which the signal for all channels and all imaging rounds were acquired. All tiles within this FOV are the same size, but the manifest allows for different spatial coordinates for different imaging rounds or channels (to accommodate slight movement between rounds, for example).
 In microscopy, a field of view corresponds to the area of the camera sensor, and many such fields of view are expected to be taken per tissue slice.
```

Comment:
I think it's more accurate to say something like "a field of view corresponds to the area of the camera sensor mapped to the sample plane" (i.e., the area of the field of view in physical coordinates is not the area of the camera sensor, but the area of the camera sensor scaled by the magnification of the microscope.)